File: algorithm/src/main/java/baekjoon/p02693/Main.java
package baekjoon.p02693;

import java.util.Arrays;
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int t = sc.nextInt();

        while (t-- > 0) {
            int[] inputs = new int[10];
            for (int i = 0; i < inputs.length; i++) {
                inputs[i] = sc.nextInt();
            }

            System.out.println(solve(inputs));
        }
    }

    private static int solve(int[] inputs) {
        Arrays.sort(inputs);
        return inputs[inputs.length - 3];
    }

}

File: algorithm/src/main/java/baekjoon/p02693/Main.kt
package baekjoon.p02693

fun main(args: Array<String>) {
    repeat(readLine()!!.toInt()) {
        val inputs = readLine()!!.split(" ").map { it.toInt() }.toIntArray()
        println(solve(inputs))
    }
}

private fun solve(inputs: IntArray): Int {
    return inputs.sorted()[inputs.size - 3]
}